solutions/devsprint-1/DeliveryAppChallenge/Components/DividerView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  DividerView.swift
+//  DeliveryAppChallenge
+//
+//  Created by Gabriel on 03/11/21.
+//
+
+import Foundation
+import UIKit
+
+public class DividerView: UIView {
```

Comment:
Como estamos trabalhando em um único módulo, não é necessário declarar essa classe como publica. Para seguir o padrão que estamos seguindo, podemos trocar para final para garantir para o compilador que não teremos herança dessa classe... O que acha? 
```suggestion
final class DividerView: UIView {
```